gangplank/ocp/bc.go

```diff
@@ -297,7 +297,12 @@ binary build interface.`)
 		}
 
 		index := n + 1
-		if err := createWorkerPod(ctx, index, eVars); err != nil {
+		cpod, err := NewCosaPodder(ctx, apiBuild, ac, pn, index)
+		if err != nil {
+			log.Errorf("FAILED TO CREATE POD DEFINITION: %v", err)
```

Comment:
Question: Why all caps?

Reply:
It's a _rage_ error. 

Actually, a problem with running in podman mode is separating out the error messages. I'll fix that up.